Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM golang:1.10.2-alpine
 
 RUN apk add --no-cache --update alpine-sdk
 
-COPY . /go/src/github.com/coreos/dex
-RUN cd /go/src/github.com/coreos/dex && make release-binary
+COPY . /go/src/github.com/dexidp/dex
```

Comment:
Probably early to update the quay references. Let's do this once we get that account setup.

I'll need your quay username too.

Reply:
added